src/tests/ftest/dfuse/simul.yaml

```
@@ -16,10 +16,11 @@ pool:
 container:
   type: POSIX
   control_method: daos
-mpi: !mux
-  openmpi:
-    mpi_type: openmpi
-    simul_path: /usr/lib64/openmpi/bin
-  mpich:
-    mpi_type: mpich
-    simul_path: /usr/lib64/mpich/bin
+mpi:
+  mpi_type: mpich
+  simul_path: /usr/lib64/mpich/bin
+client_processes:
+  np: 24
+  ppn: 12
```

Better to just specify `ppn` and let the framework set `np` based on the number of clients. This lets the test scale
```suggestion
  ppn: 12
```